Website/api/venv/model.py

```diff
@@ -8,6 +8,8 @@ def get_uuid():
 
 class User (db.Model):
```

Comment:
We probably want to also add a phone number columns for when we add text notifcications.